scripts/build/configure/termux_step_configure_meson.sh

```diff
@@ -18,6 +18,7 @@ termux_step_configure_meson() {
 		--buildtype ${_meson_buildtype} \
 		${_meson_stripflag} \
 		$TERMUX_PKG_EXTRA_CONFIGURE_ARGS \
+		"$TERMUX_PKG_EXTRA_CONFIGURE_ARGS2" \
```

Comment:
Is this for debugging?
Why couldn't these just go in `TERMUX_PKG_EXTRA_CONFIGURE_ARGS`?